packages/graph/hash_graph/lib/graph/src/knowledge/entity/mod.rs

```diff
@@ -25,12 +29,69 @@ impl fmt::Display for EntityId {
     }
 }
 
+/// An entity.
+///
+/// When expressed as JSON, this should validate against its respective entity type(s).
 #[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
 pub struct Entity {
     #[serde(flatten)]
     properties: HashMap<BaseUri, serde_json::Value>,
 }
 
+/// A record of an entity that has been persisted in the datastore, with its associated metadata.
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, Component)]
+pub struct PersistedEntity {
+    inner: Entity,
+    id: EntityId,
+    version: DateTime<Utc>,
```

Comment:
We'll see what this does in the OpenAPI spec once we generate after adding `PersistedEntity` to the output. May need a utoipa `value_type` attribute

Reply:
Have regenerated the OpenAPI Spec, are we happy with how this struct and the `PersistedEntityIdentifier` looks?